backend/backend/views.py

```diff
@@ -72,6 +64,17 @@ def get(self, request, *args, **kwargs):
 
         return ApiResponse({"tokens": [_bearer_token_dict(token, include_payload=False) for token in tokens]})
 
+    def delete(self, request, *args, **kwargs):
+        if BearerToken.objects.filter(token_id=request.GET.get("id")):
+            token = BearerToken.objects.get(token_id=request.GET.get("id"))
+            if request.user == token.user:
+                token.delete()
+                return HttpResponse("Authorized, token removed", status=200)
+            else:
+                return HttpResponse("Unauthorized, token was NOT removed", status=401)
```

Comment:
This is good, though three small notes here:
 - no need to include "authorized" in the response message since it is implied by the success
 - this is probably paranoia but let's not leak internal status to unauthorized users, just return 404 if authentication fails
 - it might be better to return a simple JSON dictionary; the reason is it's simpler to manipulate the API if it always return the same data format.
   I'm not 100% sure though, we should take a look through the rest of our code base to see whether there are other cases where we return plain text.

Reply:
I do agree that I would prefer returning a JSON file, whether or not we return plain text in other part of the code. I would more consider the other plain text to JSON than the other way around.

I think the `return HttpResponse("Unauthorized, token was NOT removed", status=401)` is more a Forbidden (403) than Unauthorized (which confusingly [relates to authentication, not authorization](https://www.rfc-editor.org/rfc/rfc9110#name-403-forbidden)) , but then I agree with @oleobal - returning 403 would expose the different tokens ID, without being sure what are the implication of it.

Reply:
I made the change except for the JSON dictionary, have you got any example in the code base ? I can't seem to find one, if not could one of you write it for let's say the 200 response ? I will figure it out for the 404